scripts/sdk-coin-generator/template/.blank.ts

```diff
@@ -0,0 +1,103 @@
+// Import coin specific libraries
+
+import {
+  BaseCoin,
+  BitGoBase,
+  checkKrsProvider,
+  getBip32Keys,
+  InitiateRecoveryOptions as BaseInitiateRecoveryOptions,
+  InvalidAddressError,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  promiseProps,
+  SignTransactionOptions as BaseSignTransactionOptions,
+  TransactionExplanation,
+  TransactionPrebuild,
+  UnexpectedAddressError,
+  VerifyAddressOptions as BaseVerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
```

Comment:
not all of these imports are used when generating Simple templates. We should probably use a different `.blank` template between the 3 types of coins we're going to generate for.